### PR TITLE
ansible-srl blog post

### PR DESCRIPTION
**Pull request by wdesmedt**

### CODE REVIEW

**Comment by wdesmedt** [comment]
@hellt  the diagrams may need fixing. Maybe better to have them in the learn-srlinux repo?

File: docs/blog/posts/2023/ansible-srlinux-1.md [hunk 1]
```diff
@@ -0,0 +1,947 @@
...
+
+It also generates a `delete` variable containing a list of configuration paths to delete when the play variable `purge=true` and when no tags are specified with the `ansible-playbook` command that would result in a partial nodal intent. It uses the node for configuration state (running configuration) that was retrieved by the `common/init` role and compares this against the nodal intent to generate the `delete` variable.
+
+Following diagram gives an overview how the low-level device intent is constructed from the various roles:
+
+<figure markdown>
+  <div class="mxgraph" style="max-width:100%;border:1px solid transparent;margin:0 auto; display:block;" data-mxgraph="{&quot;page&quot;:0,&quot;zoom&quot;:2,&quot;highlight&quot;:&quot;#0000ff&quot;,&quot;nav&quot;:true,&quot;check-visible-state&quot;:true,&quot;resize&quot;:true,&quot;url&quot;:&quot;https://raw.githubusercontent.com/wdesmedt/ansible-srl-demo/main/img/ansible-srl-intent.drawio.svg&quot;}"></div>
```
Comment (hellt) [inline docs/blog/posts/2023/ansible-srlinux-1.md:302]: @wdesmedt here is how I embed vector graphics from diagrams.net.

In this ugly div you are only concerned about three parameters:

1. **url**: at the end of this div you see the URL that is a raw github link to the original file you used in drawio editor. Make sure to point it to the right location.
1. **page**: inside the `data-mxgraph` property you see the first element that goes like `&quot;page&quot;:0`. This is html-escaped version of `page:0`. `Page:0` means that from the referenced drawio file you render first page (aka tab), they are 0-indexed.
2. **zoom**: the last property you may want to touch here is the `zoom`, which is set to `2` in this example. Zoom is in charge of making small pictures larger if desired. You can play with this param to see how it changes the diagram.

File: docs/blog/posts/2023/ansible-srlinux-1.md [hunk 2]
```diff
@@ -0,0 +1,947 @@
...
+Transformation from high-level intent to per-device low-level configuration is a one-way street. There is no way to go back from the low-level configuration to the high-level intent. This means that it is not possible to _reconcile_ changes in the network that were not driven by intent. For this to happen, a manual step is required to update the intent with the new state of the network.
+
+Finally, we would appreciate your feedback on this project. Please open an issue in the [GitHub repository][ansible-srl-demo] if you have any questions or remarks.
+
+
+[collection-doc-link]: ../../../ansible/collection/index.md
+[ansible-srl-demo]: https://github.com/wdesmedt/ansible-srl-demo
+
+
+<script type="text/javascript" src="https://viewer.diagrams.net/js/viewer-static.min.js" async></script>
```
Comment (hellt) [inline docs/blog/posts/2023/ansible-srlinux-1.md:947]: @wdesmedt fyi. This script must be included on a page where diagrams are embedded. No changes are required for this script tag.

File: docs/blog/posts/2023/ansible-srlinux-1.md (hunk 1 above)
Comment (hellt) [reply docs/blog/posts/2023/ansible-srlinux-1.md]: PS. I changed the URL escaping to a single-quotes escaping, which is much nicer now.

**Comment by hellt** [comment]
@wdesmedt thank your for your contribution.
I left a comment explaining how I add graphics and here is a color palette you can stick to to maintain visual coherence https://gitlab.com/rdodin/pics/-/wikis/colors

**Pull request merged**